lib/presentation/localization/locale_cubit.dart

```diff
@@ -0,0 +1,13 @@
+import 'dart:ui';
+
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:marvel/presentation/localization/locale_state.dart';
+
+class LocaleCubit extends Cubit<LocaleState> {
```

Comment:
Since you are using BLoC it make sence to use BLoC here as well

Reply:
done